surfactant/cmd/generate.py

```diff
@@ -313,8 +318,8 @@ def sbom(
                             # We need get_software_entry to look at the true filepath
                             filepath = true_filepath
 
-                        if install_prefix is not None:
-                            install_path = install_prefix
+                        if entry.installPrefix:
```

Comment:
I think this one might have been checking against `is not None` so that the install prefix can be set to an empty string. Maybe that intent would be clearer by adding an `or entry.installPrefix == ""` to the condition.